src/libraries/System.Net.Http/tests/FunctionalTests/DiagnosticsTests.cs

```diff
@@ -998,6 +962,112 @@ public async Task SendAsync_CustomSocketsHttpHandlerPropagator_PropagatorIsUsed(
                 });
         }
 
+        [ConditionalTheory(typeof(RemoteExecutor), nameof(RemoteExecutor.IsSupported))]
+        [InlineData(true, true, true)]      // Activity was set and ActivitySource created an Activity
```

Comment:
Would it be better to have a generated MemberData instead? 😅

Reply:
I tried to preserve some of the wisdom from the comments so it's not much shorter than the original, but it's moved in member data.